department-of-veterans-affairs/va.gov-cms
# VACMS-4655: Limit selection choice to vc.

## Description
See #4655

## Testing done
Visual, behat, phpunit

## Screenshots
N/A

## QA steps

- As an administrator go to `admin/structure/types/manage/vet_center_locations_list/fields/node.vet_center_locations_list.field_office` and visually verify that only vet center is selected

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
